src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -62,10 +63,10 @@ public ErrorResponse handleResourceExists(ResourceExistsException e) {
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT);
     }
 
-    @ExceptionHandler(OsiamBackendFailureException.class)
+    @ExceptionHandler({OsiamBackendFailureException.class, ConnectionInitializationException.class})
```

Comment:
Please generate a `503 Service Unavailable` instead of  `500 Internal Server Error`.